src/commands/database.js

```diff
@@ -8,40 +8,58 @@ const superagent = require('superagent');
 const fs = require('fs');
 const { findOwnerId } = require('../models/addon.js');
 const { resolveRealId } = require('../models/ids-resolver.js');
+const Logger = require('../logger.js');
 
 async function listBackups (params) {
 
-  const { org } = params.options;
+  const { org, format } = params.options;
   const [addonIdOrRealId] = params.args;
 
   const addonId = await resolveRealId(addonIdOrRealId);
   const ownerId = await findOwnerId(org, addonId);
 
   const backups = await getBackups({ ownerId, ref: addonId }).then(sendToApi);
 
-  if (backups.length === 0) {
+  if (backups.length === 0 && format === 'human') {
     println('There are no backups yet');
     return;
   }
 
-  const formattedLines = backups
-    .sort((a, b) => a.creation_date.localeCompare(b.creation_date))
-    .map((backup) => [
-      backup.backup_id,
-      backup.creation_date,
-      backup.status,
-    ]);
-
-  const head = [
-    'BACKUP ID',
-    'CREATION DATE',
-    'STATUS',
-  ];
-
-  println(formatTable([
-    head,
-    ...formattedLines,
-  ]));
+  switch (format) {
+    case 'json': {
+      const formattedBackups = backups.map((backup) => {
```

Comment:
Maybe we can sort on both output ?

Reply:
I am not a huge of sorting both outputs, as the current way to print those lists are in prod since a few time now. An if as me you build a parser on top of the current CLI output we will introduce a breaking change